views/EditProduct.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="stylesheets/display.style.css">
+    <link rel="stylesheet" href="stylesheets/style.css">
+    <link rel="stylesheet" href="stylesheets/bootstrap.min.css">
+    <script type="text/javascript" src="js/lib/jquery-1.11.1.min.js" ></script>
+    <script src="javascripts/jquery-1.11.3.min.js"></script>
+    <script src="javascripts/bootstrap.min.js"></script>
+    <script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.0.7/angular.min.js"></script>
+    <script src="//ajax.googleapis.com/ajax/libs/angularjs/1.2.25/angular-route.js"></script>
+    <script src="app.js"></script>
+    <script src="javascripts/nav_vert.js"></script>
+    <script src="javascripts/EditProduct.js"></script>
+    <script src="javascripts/QueryStringHelper.js"></script>
+
+    <script src="javascripts/DetailViewButtons.js"></script>
+</head>
+<div class="new-product-form">
+    <h1>Edit Product</h1>
+    <script>Edit()</script>
+    <div>
+        <strong>Editing ProductID: </strong>
+        <strong class="selectedID"></strong>
+    </div>
+    <div class="input-row">
+        <div class="input-label">Customer</div>
+        <textarea class="customer" rows="1" type="textarea"> </textarea>
+    </div>
+
+    <div class="input-row">
+        <div class="input-label">Item Name</div>
+        <textarea class="product-name" type="textarea" rows="1"></textarea>
+    </div>
+
+    <div class="input-row">
+        <div class="input-label">Description</div>
+        <textarea class="description" rows="4" cols="50"></textarea>
+    </div>
+
+    <div class="input-row">
+        <div class="input-label">Image</div>
+        <div class="upload-button">Upload Different Image</div>
```

Comment:
If we are doing an image upload we will need to create an api route that can store the image in s3. Can you add this as a todo?